### ISSUE TITLE
Media Object doesn't work (?)

### ISSUE THREAD

**Issue by MrMacStripe** [getgrav/grav#1440]
[https://learn.getgrav.org/themes/theme-vars#media-object](https://learn.getgrav.org/themes/theme-vars#media-object)

I implemented an upload field to admin->themes->mytemplate and a select field to allow the user to change his logo within the backend. 

Now I try to access the image in my template. If I'm correct the right way would be {{ media["theme://images/my-variable"].html() }} – as described in the media object link above. 

The variable is right, I can access the filename, e.g. logo.png, with {{ my-variable }} from the my-theme.yaml settings file. 

How do I display the image? 

TL;DR: I need to display a file from the theme-directory/images/filename. I can access the filename but not display the image via media[].html

**Comment by flaviocopes** [comment]
A couple problems.

- my-variable is not going to be interpreted, that way. You would need to use `"theme://images/" ~ my-variable`
- `media` does not exist, use `page.media`
- `page.media` does contain the list of files inside the page, not the ones contained in the theme
 
To fix all this, use 

```
<img src="{{ url('theme://images/' ~ my-variable) }}" />
```
